src/mappings/staking.ts

```diff
@@ -124,6 +186,7 @@ export async function handleAddDelegation(
 
   await updateTotalLock(eraManager, amountBn, 'add', indexer === source, event);
   await delegation.save();
+  await updateMaxUnstakeAmount(indexer, event);
```

Comment:
Not relate with this line, can we combine 2 `updateTotalStake` calls same as `updateTotalDelegation`.
Seems we can call `updateTotalStake` before the `delegation` check.